qemu/tests/cfg/nvdimm.cfg

```diff
@@ -134,25 +128,27 @@
                     unarmed_dimm_mem1 = on
         - nvdimm_nvml:
             no ppc64, ppc64le
-            depends_pkgs += ndctl-devel daxctl-devel pandoc cmake
+            depends_pkgs += ndctl-devel daxctl-devel cmake
             nvml_test = yes
             nvml_dir = /tmp/nvml
+            tests_dir = /tmp/nvml/src/test
             export_pmem_conf = 'export PMEMOBJ_CONF="sds.at_create=0"'
             get_nvml = "[ -d ${nvml_dir} ] && rm -rf ${nvml_dir}"
             get_nvml += ";git clone --depth=1 https://github.com/pmem/nvml.git ${nvml_dir}"
-            compile_nvml = "cd ${nvml_dir}; make"
+            compile_nvml = "cd ${nvml_dir}; make DOC=n"
```

Comment:
For nvdimm_redis test, need to update as well?

Reply:
Correct, code updated